parsons/quickbooks/quickbooks.py

```diff
@@ -0,0 +1,734 @@
+from parsons import Table
+from parsons.utilities import check_env
+from parsons.utilities.api_connector import APIConnector
+import logging
+
+# Set up logger
+logger = logging.getLogger(__name__)
+
+QB_URI = "https://rest.tsheets.com/api/v1/"
+
+
+class QuickBooks:
+    """
+    Instantiate the QuickBooks class.
+
+    TODO: Add information about the difference between accounting vs time
+
+    `Args:`
+        token: str
+            A valid QuickBooks Auth Token. Not required if ``QB_AUTH_TOKEN`` env
+            variable set. Find instructions to create yours here:
+            https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0
```

Comment:
You can use the restructured text to markup docstrings to (ie https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links). Here's an example of someone doing this in a Parsons connector: https://github.com/move-coop/parsons/blob/main/parsons/aws/aws_async.py#L30